posthog/api/app_metrics.py

```diff
@@ -74,6 +75,50 @@ def error_details(self, request: request.Request, *args: Any, **kwargs: Any) ->
         error_details = AppMetricsErrorDetailsQuery(self.team, plugin_config.pk, filter).run()
         return response.Response({"result": error_details})
 
+    def get_batch_export_runs_app_metrics_queryset(self, batch_export_id: str):
+        """Use the Django ORM to fetch app metrics for batch export runs.
+
+        Attempts to (roughly) match the following (much more readable) query:
+        ```
+        select
```

Comment:
Took me 5 minutes to write the query and like a day to struggle with the ORM.

Reply:
do we have to use the ORM? we could just use a query too can't we?